### PR TITLE
Version update v4.6.5

### PR DESCRIPTION
**Pull request by vaishaliagola27**
Version update v4.6.5

### CODE REVIEW

File: .github/workflows/create.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
   tag:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
```
Comment (ravatparmar) [inline .github/workflows/create.yml:10]: @vaishaliagola27  This has been tested?

File: app/assets/js/rtMedia.backbone.js [hunk 2]
```diff
@@ -46,6 +46,31 @@ jQuery( function( $ ) {
...
 			imageEdit.save( $media_id, $nonce );
 		} );
+
+		/**
+		 * Reload page when rtmedia_update type of activity is edited.
+		 */
+		function filterBeaSaveSuccess() {
+			location.reload();
+		}
+		/**
```
Comment (ravatparmar) [inline app/assets/js/rtMedia.backbone.js:56]: Can we add one blank line between two function calls.

File: app/assets/js/rtMedia.backbone.js [hunk 3]
```diff
@@ -46,6 +46,31 @@ jQuery( function( $ ) {
...
+				return;
+			}
+
+			if ( ! $( '#activity-' + originalOptions.data.activity_id ).hasClass( 'rtmedia_update' ) ) {
+				return;
+			}
+
+			// Change the callback function to our own function, which reloads the page.
+			originalOptions.success = filterBeaSaveSuccess;
+			options.success = filterBeaSaveSuccess;
```
Comment (ravatparmar) [inline app/assets/js/rtMedia.backbone.js:72]: Can we align 'equal to' here

File: tests/codeception/composer.json [hunk 4]
```diff
@@ -1,5 +1,7 @@
 {
     "require-dev": {
-        "lucatume/wp-browser": "*"
+        "lucatume/wp-browser": "*",
+        "codeception/module-webdriver": "^1.1",
+        "codeception/module-asserts": "^1.3"
```
Comment (ravatparmar) [inline tests/codeception/composer.json:5]: @vaishaliagola27  This also has been tested, right?

File: app/assets/js/rtMedia.backbone.js [hunk 5]
```diff
@@ -1443,16 +1468,29 @@ jQuery( document ).ready( function( $ ) {
...
+					object = 'group';
+				} else {
+					object = 'profile';
+				}
 			} else {
-				object = 'profile';
+				var whatsNewPostIn = jQuery( '#whats-new-post-in' );
+				if ( whatsNewPostIn.length ) {
+					object = whatsNewPostIn.val();
+					item_id = 0;
```
Comment (ravatparmar) [inline app/assets/js/rtMedia.backbone.js:1487]: align here as well.

File: tests/codeception/composer.json (hunk 4 above)
Comment (pooja-muchandikar) [reply tests/codeception/composer.json:5]: @ravatparmar Yes

**Review by ravatparmar — APPROVED** [review]

**Pull request merged**